python313Packages.gradio-client: 1.7.2 -> 1.10.0

pbsds opened the pull request:
- **python313Packages.gradio-client: 1.7.2 -> 1.8.0**
- **python313Packages.gradio: 5.20.0 -> 5.22.0**

~~To get gradio to build I had to apply a [dirty hack](https://gist.github.com/pbsds/475cfae99e56784c5d1aea7be6954624) to pydub, due to some error in ffmpeg-full which I believe https://github.com/NixOS/nixpkgs/pull/392115 could have played a part in. This build failure have yet to become visible on hydra however, so it may be an issue on my infra~~


closes https://github.com/NixOS/nixpkgs/pull/392452

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

pbsds committed:
python313Packages.gradio-client: 1.7.2 -> 1.8.0

Changelog: https://github.com/gradio-app/gradio/releases/tag/gradio_client@1.8.0

pbsds committed:
python313Packages.gradio: 5.20.0 -> 5.22.0

Changelog: https://github.com/gradio-app/gradio/releases/tag/gradio@5.22.0

FliegendeWurst reviewed (approved):
The ffmpeg-full issue has been fixed

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 392559`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 20 packages built:</summary>
  <ul>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio-client</li>
    <li>python312Packages.gradio-client.dist</li>
    <li>python312Packages.gradio-pdf</li>
    <li>python312Packages.gradio-pdf.dist</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python312Packages.smolagents</li>
    <li>python312Packages.smolagents.dist</li>
    <li>python313Packages.gradio</li>
    <li>python313Packages.gradio-client</li>
    <li>python313Packages.gradio-client.dist</li>
    <li>python313Packages.gradio-pdf</li>
    <li>python313Packages.gradio-pdf.dist</li>
    <li>python313Packages.gradio.dist</li>
    <li>python313Packages.smolagents</li>
    <li>python313Packages.smolagents.dist</li>
  </ul>
</details>

FliegendeWurst commented:
I think you need to force-push with a no-op change (e.g. `commit --amend` without changes).

FliegendeWurst commented:
Timeout on aarch64-darwin, others pass

FliegendeWurst commented:
Hmm... I tried building it on the community aarch64-darwin builder and it got stuck somewhere, the build is still running. So it isn't just ofborg.

pbsds committed:
python313Packages.gradio: 5.22.0 -> 5.28.0

Changelog: https://github.com/gradio-app/gradio/releases/tag/gradio@5.28.0

pbsds committed:
python313Packages.gradio-client: 1.8.0 -> 1.10.0

Changelog: https://github.com/gradio-app/gradio/releases/tag/gradio_client@1.10.0

pbsds commented:
done in https://github.com/NixOS/nixpkgs/pull/404958
